### PR TITLE
Added Pijul

### PR DESCRIPTION
**Pull request by vRobM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ Table of Contents
   * [github.com](https://github.com/) — Unlimited public repositories and unlimited private repositories (up to 3 collaborators) with Pipelines for CI/CD
   * [bitbucket.org](https://bitbucket.org/) — Unlimited public and private Git repos for up to 5 users with Pipelines for CI/CD
   * [gitlab.com](https://about.gitlab.com/) — Unlimited public and private Git repos with unlimited collaborators and 10 GB space per repo
+  * [pijul.com](https://pijul.com/) - Unlimited free and open source distributed version control system. Its distinctive feature is to be based on a sound theory of patches, which makes it easy to learn and use, and really distributed. Solves many problems of git/hg/svn/darcs.
```
Comment (ripienaar) [inline README.md]: this doesnt appear to be a service? it's a tool you download but hosting and so forth is up to you?

Comment (vRobM) [reply README.md]: it's both, like github, you can self host or use the public service.

Comment (ripienaar) [reply README.md]: ah didnt see the nest thing, ok

**Comment by ripienaar** [comment]
thanks a lot

**Pull request merged**